substra/sdk/client.py

```diff
@@ -764,3 +764,7 @@ def leaderboard(self, objective_key, sort='desc'):
         """Get objective leaderboard"""
         return self.client.request('get', assets.OBJECTIVE, f'{objective_key}/leaderboard',
                                    params={'sort': sort})
+
+    def cancel_compute_plan(self, compute_plan_id):
+        """Cancel execution of compute plan."""
+        return self.client.get(assets.COMPUTE_PLAN, compute_plan_id)
```

Comment:
This will get a compute plan and not cancel an existing one. 
A new route will be created in the backend to handle the compute plan execution. The route will be something like `computeplan/<id>/cancel`.

Reply:
Okay, but I'm not sure I can change that until this is done in the backend?

Reply:
It has been indeed not defined in the specification, the PR for the backend is now available:
https://github.com/SubstraFoundation/substra-backend/pull/65
It may change in the future after the review but you can start with: `POST /computeplan/<id>/cancel`